cuvslam-skills/cuvslam-onboard/references/dataset-guides.md

```
@@ -0,0 +1,227 @@
+{
+  "fps": 10,
+  "sequence": [["00/00.0.0001.png"], ["01/00.1.0001.png"]]
+}]
+```
+
+---
+
+## Converting ROS Bags
+
+**bag2edex** (offline, frame-by-frame):
+```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)